setup.py

```diff
@@ -16,7 +16,7 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    'html5lib>=0.99999999,>=1.0b9',
```

Comment:
This felt wrong, so I spent some time looking for the specification that dictates how dependency versions are specified.

https://www.python.org/dev/peps/pep-0440/#version-specifiers

According to that, the `,` denotes an AND between version clauses. Thus this change says "use html5lib >= 0.99999999 AND >= 1.0b9" which doesn't make any sense.

Seems like the more appropriate way to do this is something like:

```
html5lib>=0.99999999,!=1.0b1,!=1.0b2,!=1.0b3,!=1.0b4,!=1.0b5,!=1.0b6,!=1.0b7,!=1.0b8
```

That says "html5lib >= 0.99999999 and, oh, by the way, don't allow this set of known-bad versions".